script/goal/MoveEast.js

```diff
@@ -8,7 +8,7 @@ export default class MoveEast extends Goal {
 	}
 
 	takeAction(actor) {
-		return new Promise((success, fail) => {
+		return new Promise((succeed, fail) => {
```

Comment:
Nog geen async/await? 😄

Reply:
Dit is helemaal niet async of wel? Interessant dat je wel een promise gebruikt. Komt zeker goed uit qua error handling? Je kunt ook exceptions gooien als alternatief!

Reply:
Ik gebruik `async/await` inmiddels wel meer, onder andere voor turn resolution. Ik vond voor die goals dit uiteindelijk de meest leesbare interface, maar mogelijk is het wel wat overengineered; dit kan ik natuurlijk ook oplossen met een boolean return.

Reply:
Dit is het enige moment dat het relevant is dat dit een promise returned tot dusver: https://github.com/joeribakker/sorcerer/blob/master/script/module/Brain.js#L27

En dat kan volgens mij ook gewoon met een boolean return opgelost worden.